github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -135,6 +135,7 @@ presubmits:
       fork-per-release: "true"
     always_run: false
```

Comment:
Do you want always_run too?

Reply:
Yes, ofcourse :(

kubevirt/project-infra#332